public-repositories/published-repositories.json

```diff
@@ -0,0 +1,310 @@
+[
+    {
+        "name": "ICRC",
+        "email": "",
+        "organization": "dfinity",
+        "critical": false,
```

Comment:
This needs to be clearly explained what does this field mean.

Reply:
> Having two lists will confuse make changes more complicated. Why not prioritize cla bot to workflow migration instead to have all the code in the single place?

I'm okay with prioritizing the cla bot migration before this. Will address the comments when I pick this up again.